Example/PaymentSheet Example/PaymentSheet Example/Appearance+Codable.swift

```diff
@@ -0,0 +1,348 @@
+        self.embeddedPaymentElement.row.flat.bottomSeparatorEnabled = try container.decode(Bool.self, forKey: .embeddedFlatBottomSeparatorEnabled)
+
+        // Radio colors
+        if let selectedColor = try container.decodeIfPresent(CodableUIColor.self, forKey: .embeddedFlatRadioSelectedColor) {
+            self.embeddedPaymentElement.row.flat.radio.selectedColor = selectedColor.uiColor
+        }
+        if let unselectedColor = try container.decodeIfPresent(CodableUIColor.self, forKey: .embeddedFlatRadioUnselectedColor) {
+            self.embeddedPaymentElement.row.flat.radio.unselectedColor = unselectedColor.uiColor
+        }
+
+        // Checkmark and chevron colors
+        if let checkmarkColor = try container.decodeIfPresent(CodableUIColor.self, forKey: .embeddedFlatCheckmarkColor) {
+            self.embeddedPaymentElement.row.flat.checkmark.color = checkmarkColor.uiColor
+        }
+        self.embeddedPaymentElement.row.flat.chevron.color = try container.decode(CodableUIColor.self, forKey: .embeddedFlatChevronColor).uiColor
+
+        // Floating properties
+        self.embeddedPaymentElement.row.floating.spacing = try container.decode(CGFloat.self, forKey: .embeddedFloatingSpacing)
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+
+        // Top-level properties
+        try container.encode(cornerRadius, forKey: .cornerRadius)
+        try container.encode(borderWidth, forKey: .borderWidth)
+        try container.encodeIfPresent(selectedBorderWidth, forKey: .selectedBorderWidth)
+        try container.encode(sheetCornerRadius, forKey: .sheetCornerRadius)
+        try container.encode(sectionSpacing, forKey: .sectionSpacing)
+        try container.encode(verticalModeRowPadding, forKey: .verticalModeRowPadding)
+
+        let iconStyleString =
+        switch iconStyle {
+        case .filled: "filled"
+        case .outlined: "outlined"
+        default: throw AppearanceCodableError(description: "Unknown icon style")
+        }
+        try container.encode(iconStyleString, forKey: .iconStyle)
+
+        try container.encode(textFieldInsets, forKey: .textFieldInsets)
+        try container.encode(formInsets, forKey: .formInsets)
+
+        // Font properties - will need to be expanded if more options are added to the playground
+        try container.encode(font.sizeScaleFactor, forKey: .fontSizeScaleFactor)
+        let fontBaseDescriptorData = try NSKeyedArchiver.archivedData(withRootObject: font.base.fontDescriptor, requiringSecureCoding: false)
+        try container.encode(fontBaseDescriptorData, forKey: .fontBaseDescriptor)
+        if let headline = font.custom.headline {
+            let headlineFontDescriptorData = try NSKeyedArchiver.archivedData(withRootObject: headline.fontDescriptor, requiringSecureCoding: false)
```

Comment:
I generally wouldn't recommend this in production (especially without `requiringSecureCoding`!) but it seems fine here